### PR TITLE
Parallax practice assignment

### LINKED ISSUE #1: Day 7 Take Home Challenge: Recreate parallax practice mockup
**To-do:**
- [x] Create new branch named `take-home`
- [x] Add two fonts from Google Fonts
- [x] Add `img` folder with background images
- [x] Add navbar with links to all page sections
- [x] Add jumbotron
- [x] Add `main` content
- [x] Add parallax scrolling effect to all sections
- [x] Add `footer` with social media links
- [x] Change text to reflect mockup
- [x] Add buttons between sections
- [x] Fix text box styling to match mockup
- [x] Adjust styling for smaller screens
- [x] Add, commit, and push changes to `take-home`
- [x] Open a PR
- [x] Request review from collaborator

### PR DESCRIPTION
**Pull request by im-amy-lew**
**Created page based off of mockup shown below featuring the parallax scrolling effect:**

parallax-practice

Also includes layout changes for mobile screens:
image
image

Closes #1.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+* {
+  font-family: var(--body-font);
+}
+
+:root {
+  --jumbotron-font: 'Lobster', cursive;
+  --body-font: 'Open Sans', sans-serif;
+}
```
Comment (ayojackay) [inline css/styles.css:8]: I love that you're using CSS selectors! They're so helpful!

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.text_main {
+  color: var(--bs-gray-400);
+}
+
+.border {
+  border-color: var(--bs-gray-700) !important;
+  padding: 0.75rem 1.7rem;
+}
```
Comment (ayojackay) [inline css/styles.css:81]: Two things:
- Great job utilizing BS selectors that are provided as suggestions in the documentation
- Using `!important` when necessary, but not too much xD

**Review by ayojackay — APPROVED** [review]
Wonderful job, Amy! I left a few comments on things that I saw shine out to me. Continue to research and use the documentation. Documentation is vital when learning about new tools, concepts, and more! I appreciate you doing the assignment!

**Comment by im-amy-lew** [comment]
Thank you so much for the feedback @ayojackay! ❤️

**Pull request merged**